machines/settings.ts

```diff
@@ -126,6 +129,20 @@ export const settingsMachine = model.createMachine(
           },
         },
       },
+      injiToutGuide: {
```

Comment:
@Anil-kumar-Majji state should be "injiTourGuide" not "injiToutGuide". Please make this change and push the code

Reply:
Done